gradle/publish.gradle

```diff
@@ -101,34 +99,19 @@ publishing {
   See the License for the specific language governing permissions and
   limitations under the License.
   ''')
-            elem.insertBefore(hdr, elem.firstChild)
-          }
+          elem.insertBefore(hdr, elem.firstChild)
         }
       }
     }
   }
   repositories {
     maven {
-      if (project.hasProperty("mavenRepository")) {
-        url = project.mavenRepository
-      } else if (project.isReleaseVersion) {
-        url = "https://repository.apache.org/service/local/staging/deploy/maven2"
-      } else if (project.hasProperty("mavenSnapshotBucket")) {
-        // If testing from a non-GCE instance, then the shell needs service-account creds
-        // following the instructions at https://cloud.google.com/docs/authentication/production
-        url = "gcs://$project.mavenSnapshotBucket/snapshots"
-      } else {
-        url = "gcs://maven.apachegeode-ci.info/snapshots"
-      }
-      if (!url.toString().startsWith("gcs:")) {
```

Comment:
You may still need this if statement to avoid GCS handler trying to use credentials. I feel like this was an issue.

Reply:
If you put a `println credentials` before and after the `credentials` declaration, both report `[username: null]`, so it appears that the default credential set is a singleton with a null `username`.  I'm not sure if this was different in Gradle4, though -- I would also be unsurprised by an emptyset of credentials or a singleton with a `null` rather than a credential object with a `null` username and password.